errdefs/http_helpers.go

```diff
@@ -76,26 +68,50 @@ func GetHTTPErrorStatusCode(err error) int {
 	return statusCode
 }
 
-func apiVersionSupportsJSONErrors(version string) bool {
-	const firstAPIVersionWithJSONErrors = "1.23"
-	return version == "" || versions.GreaterThan(version, firstAPIVersionWithJSONErrors)
-}
+// FromStatusCode creates an errdef error, based on the provided status-code
```

Comment:
If not in the name of the function, at least mention in the comment that these are HTTP status codes.